src/components/admin/AppLayout.vue

```diff
@@ -1,19 +1,24 @@
 <template>
```

Comment:
![image](https://user-images.githubusercontent.com/5394573/63714481-8ec63400-c84a-11e9-85af-e047c6c845fc.png)

Reply:
Leaps and jumps.

![2019-08-26_21-45-23](https://user-images.githubusercontent.com/5394573/63714703-02684100-c84b-11e9-9f28-2006889fd69a.gif)